workers/unity/Packages/io.improbable.gdk.deploymentlauncher/DeploymentLauncherWindow.cs

```diff
@@ -240,21 +240,6 @@ private void OnGUI()
                     EditorGUILayout.LabelField("Project Name", projectName);
                 }
 
-                using (new EditorGUILayout.HorizontalScope())
-                {
-                    using (new EditorGUI.DisabledScope(manager.IsActive))
-                    {
-                        if (GUILayout.Button(style.EditRuntimeVersionButtonContents, EditorStyles.miniButton, GUILayout.ExpandWidth(false)))
-                        {
-                            GdkToolsConfigurationWindow.ShowWindow();
```

Comment:
I think you can restore this functionality with https://docs.unity3d.com/2019.3/Documentation/ScriptReference/SettingsService.OpenProjectSettings.html

Reply:
Ohh, I didn't realise this was the deployment launcher settings! I should have looked at the filename 🤦‍♂️